src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -16,14 +16,25 @@
     /** The list of persons that was produced by the command */
     private final List<? extends ReadOnlyPerson> relevantPersons;
 
+    public final Boolean lite;
```

Comment:
Class variables should never be declared public as recommended by the coding standard. Plus, the name of the variable is not very informative as to what it is referring to.